Automattic/woocommerce-payments
# [Draft] Add JCB payment method

Fixes #6804

#### Changes proposed in this Pull Request
Add a JCB payment method to the payment methods list on the settings page.

<img width="485" alt="Screenshot 2023-08-18 at 17 08 40" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/9f7657fc-5dc4-4d48-84f1-09f8ce08aeb2">

#### Testing instructions
- Connect a Japanese account.
- Navigate to `Payments` -> `Settings`.
- Verify the JCB payment method is displayed.

-------------------


**Post merge**
